docs/tutorial/index.md

```diff
@@ -3,37 +3,71 @@ title: Gatsby.js Tutorials
-8.  [Preparing a site to go live](/tutorial/part-eight/): Learn how to audit your site for performance and best practices for accessibility, SEO, and more.
+4. [Abfragen von Daten in einem Blog](/tutorial/part-four/): Erstellung eines
+   Blogs und Nutzung einer GraphQL Anfrage um deinen Webseiten Titel in die
+   Kopfzeile des Blogs zu ziehen.
+5. [Datenquellen Plugins und Darstellung von angefragten Daten](/tutorial/part-five/):
+   Nutzen eines Datenquellen Plugins um Markdown Blog Beiträge in deine Seite
+   zu ziehen und Erstellung einer Index Seite mit der Liste aller Blog Beiträge.
+6. [Umformende Plugins](/tutorials/part-six/): Nutzung eines umformenden
+   Plugins um Markdown Blog Beiträge in eine Form umzuwandeln, die der Blog
+   darstellen kann.
+7. [Programatisches Erstellen von Seiten aus Daten](/tutorial/part-seven/):
+   Lerne wie Seiten für deinen Blog programatisch erstellt werden können.
+8. [Vorbereiten einer Seite zum Veröffentlichen](/tutorial/part-eight/):
+   Lerne wie du deine Seite auf Performance und empfohlene Vorgehensweisen
+   für Barrierefreiheit, SEO und mehr prüfen kannst.
 
-## Theme tutorials
+## Themen Tutorials
 
-Gatsby themes abstract all of your default configuration (shared functionality, data sourcing, design) out of your site and into an installable package.
+Gatsby Themen abstrahieren alle Standardeinstellungen (geteilte Funktionalität,
+Datenquellen, Design) aus deiner Seite und in ein installierbares Paket.
 
-This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
+Das bedeutet das die Konfiguration und Funktionalität nicht direkt in deinem
+Projekt fest geschrieben ist, sondern als abhängiges Paket versioniert, zentral
+verwaltet und installiert wird. Du kannst problemlos ein Thema aktualiseren,
+Themen zusammensetzen oder eines für ein anderes austauschen.  
 
-Learn more about using and building themes with the [Gatsby theme tutorials](/tutorial/theme-tutorials/).
+Lerne mehr über die Nutzung und das Erstellen von Themen mit dem
+[Gatsby Themen Tutorial](/tutorial/theme-tutorials/).
 
-## Additional tutorials
+## Zusätzliche Tutorials
 
-There are many use cases for Gatsby, some of which aren't covered in the first set of tutorials to keep you focused while learning. Gatsby's Additional Tutorials section is a collection of resources showing how to complete even more Gatsby tasks step-by-step, such as using source plugins for images and CMS content.
+Es gibt viele Nutzungsszenarien für Gatsby, einige werden nicht von dieser
+ersten Reihe von Tutorials abgedeckt um dich beim Lernen fokussiert zu halten.
+Gatsbys zuätzliche  Tutorials Sektion ist eine Sammlung von Anleitungen, die
```

Comment:
- `zuätzliche` should be `zusätzliche`
- replace the two spaces with one

Reply:
Thanks, must have slipped through